### PR TITLE
add first draft of structuring interviews tutorial

### PR DESCRIPTION
**Pull request by leah-ngaari**
This tutorial takes design thinkers through a step by step guide on how to structure interviews.

### CODE REVIEW

**Comment by n-orlowski** [comment]
I think this should live within the tutorials section and we could have a template version that lives in the design system to link to.. Or we can create a new section in the Design System specific for Service Designers. Thoughts @smbuthia  ?

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Interviews can be a great way to empathize with your users because interviews can give you an in-depth understanding of the users’ values, perceptions, and experiences. They allow you to ask specific questions, while remaining open to exploring your participants’ points of view. They are also often combined with other user research methods, such as usability tests or surveys, so as to gain deeper insights into objective results by asking a user about them and to elicit the user’s subjective opinion on products or interactions.
+
+## Objectives
+
+- To understand pros and cons of semi structured qualitative interviews
+- To understand when to conduct user interviews
+- To understand how to structure user interviews
+
+## Expected outcome
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:28]:
```suggestion
## Expected Outcome
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+By the end of the tutorial design thinkers will:
+
+- Gain knowledge on user interviews
+- Apply user interview best practices in their field of work
+
+## Brief overview of key concepts
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:35]:
```suggestion
## Key Concepts
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+- This means you have an interview guide with the questions or themes that you want to talk to the user about, but you are free to change the order of questions or to explore different topics that may arise during the interview. 
+
+- User interviews can be very informative and helpful, but only if they are used correctly and for the right things. It’s important to know what you can expect to get out of interviews and what you shouldn’t expect to get out of interviews.
+
+- User interviews can be conducted for the following reasons:
+  - Exploration-before a clear concept has been defined or before a major redesign.
+  - In combination with user tests and formal experiments
+  - In combination with observations
+
+## User Interview requirements
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:48]:
```suggestion
## User Interview Requirements
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+
+- A good interview requires preparation and careful consideration on the part of the interviewer.
+- It’s important to be aware of how to ask questions and how to listen in order to gain valid insights into your participant’s life and experiences
+
+## How to structure user interviews
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:53]:
```suggestion
## User Interview Structure
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 5]
```diff
@@ -0,0 +1,86 @@
...
+
+When you conduct a user interview, it is important to consider the best way of ordering questions. Below are some tips on what to ask in the beginning, in the middle, and at the end of a user interview. 
+
+## Beginning
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:57]:
```suggestion
### Beginning
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 6]
```diff
@@ -0,0 +1,86 @@
...
+
+In the beginning of the interview, you should ask opening questions to set people at ease and build rapport. You should not ask about sensitive topics. Instead, focus on setting the stage and bringing your interviewees on board so they’re comfortable enough to be, and remain, open with you. Your questions  at this stage can include the following:
+- Brief participants what topics will be covered
+- Brief participants how their data will be used
+- Asking “safe” questions such as what their role is in their organization
+- Asking concrete questions that are easy to answer
+
+## Middle
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:65]:
```suggestion
### Middle
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 7]
```diff
@@ -0,0 +1,86 @@
...
+
+In the middle of the interview, you’re hitting high gear. Having established a direct channel with your interviewees (the users), you move on and ask the bulk of your questions. The predetermined order of the questions may change based on the direction the conversation is taking. Some of the tips here include:
+- Keep the flow of the conversation as natural as possible, but cover the topics you want to cover.
+- Pick up on what participants have said earlier and get full replies to questions they have only partially answered.
+- Steer participants back on track if they go too far off topic.
+- Show that you have been listening. 
+
+## End
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:73]:
```suggestion
### End
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 8]
```diff
@@ -0,0 +1,86 @@
+---
+title: "Empathizing with users"
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:2]:
```suggestion
title: "Empathizing With Users"
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 9]
```diff
@@ -0,0 +1,86 @@
...
+linkTitle: Structuring user interviews
+weight: 1
+description: >
+  How to structure user interviews
+relatedContent: >
+  design/personas
+---
+
+
+{{% pageinfo %}}Understanding the people you are designing for is a paramount step in the design process, and so one way to achieve that is through conducting user interviews. This tutorial will help design thinkers to understand users’ experiences from their own point of view in terms of:
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:12]:
```suggestion
{{% pageinfo %}}Understanding the people you are designing for is a paramount step in the design process. One way to achieve that is through conducting user interviews. This tutorial will help design thinkers to understand users’ experiences from their own point of view in terms of: 
```

Comment (n-orlowski) [reply content/en/design/apps/tutorials/structuring-interviews.md:12]: Is there a clearer term than "design thinkers"? App designers, perhaps?

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 10]
```diff
@@ -0,0 +1,86 @@
...
+
+{{% pageinfo %}}Understanding the people you are designing for is a paramount step in the design process, and so one way to achieve that is through conducting user interviews. This tutorial will help design thinkers to understand users’ experiences from their own point of view in terms of: 
+
+- Why they use products in a certain way
+- How they feel about something
+- How they perform various actions{{% /pageinfo %}}
+
+## Background (current situation, problem, and/or opportunity)
+
+Interviews can be a great way to empathize with your users because interviews can give you an in-depth understanding of the users’ values, perceptions, and experiences. They allow you to ask specific questions, while remaining open to exploring your participants’ points of view. They are also often combined with other user research methods, such as usability tests or surveys, so as to gain deeper insights into objective results by asking a user about them and to elicit the user’s subjective opinion on products or interactions.
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:20]:
```suggestion
Interviews can be a great way to empathize with your users as they can provide an in-depth understanding of a users’ values, perceptions, and experiences. They allow you to ask specific questions while remaining open to exploring your participants’ points of view. They are also often combined with other user research methods, such as usability tests or surveys to gain deeper insights.
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 11]
```diff
@@ -0,0 +1,86 @@
...
+
+## Objectives
+
+- To understand pros and cons of semi structured qualitative interviews
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:24]:
```suggestion
- To understand pros and cons of semi-structured qualitative interviews
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 12]
```diff
@@ -0,0 +1,86 @@
...
+- To understand when to conduct user interviews
+- To understand how to structure user interviews
+
+## Expected outcome
+
+By the end of the tutorial design thinkers will:
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:30]: Same comment re: "design thinkers" as above

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 13]
```diff
@@ -0,0 +1,86 @@
...
+
+- Gain knowledge on user interviews
+- Apply user interview best practices in their field of work
+
+## Brief overview of key concepts
+
+- As the word implies, semi-structured interviews are somewhat structured in that you prepare a set of topics you would like to cover during the interview, but still open enough that you can follow leads in the conversation and change the order of topics.
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:37]:
```suggestion
- As the term implies, semi-structured interviews are somewhat structured in that you prepare a set of topics you would like to cover during the interview, but that are still flexible enough that you can follow leads in the conversation and change the order of topics.
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 14]
```diff
@@ -0,0 +1,86 @@
...
+
+- This means you have an interview guide with the questions or themes that you want to talk to the user about, but you are free to change the order of questions or to explore different topics that may arise during the interview.
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:39]: Not sure we need this, seems repetitive from the previous sentence

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 15]
```diff
@@ -0,0 +1,86 @@
...
+## Brief overview of key concepts
+
+- As the word implies, semi-structured interviews are somewhat structured in that you prepare a set of topics you would like to cover during the interview, but still open enough that you can follow leads in the conversation and change the order of topics.
+
+- This means you have an interview guide with the questions or themes that you want to talk to the user about, but you are free to change the order of questions or to explore different topics that may arise during the interview. 
+
+- User interviews can be very informative and helpful, but only if they are used correctly and for the right things. It’s important to know what you can expect to get out of interviews and what you shouldn’t expect to get out of interviews.
+
+- User interviews can be conducted for the following reasons:
+  - Exploration-before a clear concept has been defined or before a major redesign.
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:44]:
```suggestion
  - Exploration – before a clear concept has been defined or before a major redesign.
```

File: content/en/design/apps/tutorials/structuring-interviews.md [hunk 16]
```diff
@@ -0,0 +1,86 @@
...
+  - In combination with user tests and formal experiments
+  - In combination with observations
+
+## User Interview requirements
+
+- A good interview requires preparation and careful consideration on the part of the interviewer.
```
Comment (n-orlowski) [inline content/en/design/apps/tutorials/structuring-interviews.md:50]: In the bulleted lists some items have periods at the end and some don't. I would reco to not use them to be consistent with the rest of the docs :)

**Comment by smbuthia** [comment]
>I think this should live within the tutorials section and we could have a template version that lives in the design system to link to.. Or we can create a new section in the Design System specific for Service Designers. 

I think having a new section in the Design System would make it easier for Service Designers to find and contribute to the content.

File: content/en/design/apps/tutorials/structuring-interviews.md (hunk 16 above)
Comment (leah-ngaari) [reply content/en/design/apps/tutorials/structuring-interviews.md]: Updated

**Pull request closed without merging**